Makefile

```diff
@@ -91,3 +91,13 @@ dev-config-restore:
 dev-update-source:
 	rsync -av $(SOURCE_PACKAGE)/bin/ $(OUTPUT_PACKAGE)/bin/
 	rsync -av $(SOURCE_PACKAGE)/default/ $(OUTPUT_PACKAGE)/default/
+
+dev-install-dependencies-pack:
+	pip install --upgrade-strategy only-if-needed -r requirements-pack.txt
+	sudo pip install --upgrade-strategy only-if-needed -r requirements-pack-sudo.txt
+
+dev-install-dependencies-lib:
+	pip install --upgrade-strategy only-if-needed -r TA_dataset/lib/requirements.txt
+
+dev-install-dependencies-dev:
```

Comment:
where is dev-install-dependencies-pack-sudo ?

Reply:
Good catch :)